app/src/main/java/org/jellyfin/androidtv/util/InfoLayoutHelper.java

```diff
@@ -106,6 +106,10 @@ public static void addInfoRow(Context context, BaseItemDto item, int mediaSource
         addSeriesStatus(context, item, layout);
         addRatingAndRes(context, item, mediaSourceIndex, layout);
         addMediaDetails(context, audioStream, layout);
+        if (Utils.isTrue(item.getHasSubtitles())) {
+            addBlockText(context, layout, "CC");
+            addSpacer(context, layout, "  ");
+        }
```

Comment:
We should prioritize the subtitle information before the audio information. That's the way the web client also does it.

Reply:
Apple puts it [after](https://help.apple.com/assets/6414D1E8F0A05F36C35A15D0/6414D1EBF0A05F36C35A15DA/en_GB/9b152618525e549cad25615d6619e2b4.png) the audio. Amazon puts it [before](https://i0.wp.com/www.aftvnews.com/wp-content/uploads/2022/09/The-Lord-of-the-Rings-The-Rings-of-Power-on-Prime-Video.jpg). It looks very awkward sandwiched between the two. It's also the only optional element so the layout is not consistent for different items - that's why I put it at the end. A/V info should be next to each other IMO so it's easy to read.

Reply:
I moved it before the video information. It looks better next to the rating before all the technical stuff.